docs/source/requirements.txt

```diff
@@ -0,0 +1,16 @@
+Sphinx
```

Comment:
These are the requirements to build the docs?

Could we fix the versions? Just so that there aren't any issues when trying to reproduce building the docs

Reply:
Sure, I was going to do it but then I had already removed the venv I created :)

Reply:
Done